ReSharper.FSharp/test/data/features/quickFixes/ignoreUnusedBindingExpression/Multiline 01 - Seq expr.fs.gold

```diff
@@ -0,0 +1,7 @@
+﻿module Module
+
+let a =
+  1{caret}
+  1 |> ignore
+
+  ()
```

Comment:
Please make sure it's a correct sequence expression node after modification, i.e. it should be something like
```
seqExpr
  1
  1 |> ignore
  ()
```
and not like
```
seqExpr
  seqExpr
    1
    1 |> ignore
  ()
```
It'd be great to have this logic reused in InlineVar refactoring later.

We should probably dump the resulting tree in some of the tests with modifications.